addons/overheating/functions/fnc_jamWeapon.sqf

```diff
@@ -43,7 +43,7 @@ if (_jamTypesAllowed isEqualTo []) then {
     for "_i" from count _jamTypesAllowed -2 to 0 step -2 do {
         private _jamCurretType = toLowerANSI (_jamTypesAllowed select _i);
         if !(_jamCurretType in ["eject", "extract", "feed", "fire", "dud"]) exitWith { // check config values and switch to default values if unusual value found
-            ERROR_2("Weapon '%1' has unexpected value %2 in QQGVAR(jamTypesAllowed). Expected values are 'Eject', 'Extract', 'Feed', 'Fire', 'Dud'.",_weapon,_jamCurretType);
+            ERROR_3("Weapon '%1' has unexpected value %2 in '%3'. Expected values are 'Eject', 'Extract', 'Feed', 'Fire', 'Dud'.",_weapon,_jamCurretType,QGVAR(jamTypesAllowed));
```

Comment:
This was intential to put the name of the config entry into the message, rather than the array contents which this change does.

Reply:
I understand the intent, I just changed it so that the macro is expanded outside of the string, which in then formatted into said string at run time. I'm not inserting the contents of the array, I'm inserting the name.

Reply:
Ahh right, carry on!